src/Pagination/ScrollerInterface.php

```diff
@@ -10,21 +10,12 @@
 
 interface ScrollerInterface
 {
-    /**
-     * @param RestClient $client
-     * @param JobConfig $jobConfig
-     * @return RestRequest|false
-     */
-    public function getFirstRequest(RestClient $client, JobConfig $jobConfig);
+    public function getFirstRequest(RestClient $client, JobConfig $jobConfig): ?RestRequest;
```

Comment:
Toto tu bolo trocha nestastne vyriesene, vracalo to `RestRequest|false` a robilo to problemy (phpstan, ...), kedze su to rozne datove typy. Fixnute na ``RestRequest|null`.